performance-tracking-core/src/test/java/com/rakuten/tech/mobile/perf/core/MeasurementBufferSpec.java

```diff
@@ -34,38 +25,84 @@
   }
 
   @Test public void shouldProvideNullWhenBufferIsFull() {
-    for (int i = 0; i < buffer.at.length; i++) {
-      buffer.next();
-    }
+    fillBuffer(buffer, 512);
```

Comment:
I think all the `512`s should be `MeasurementBuffer.SIZE` instead, no?

- [ ] use `MeasurementBuffer.SIZE` in tests